src/helpers/ApiCallHelper.tsx

```
@@ -1,5 +1,21 @@
+    return fetch(`${process.env.REACT_APP_BASE_URL}/v1/stations`, {
+        headers: {
+            'X-API-KEY': `${process.env.REACT_APP_X_API_KEY}`,
+        },
+    });
+};
+
+export type getParams = ([string, string])[];
+
+export const makeGetRequestWithParams = (route: string, params: getParams) => {
+    const paramsString = params.map(([key, value]) => {
+        return encodeURI(key) + '=' + encodeURI(value);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)